src/box/lua/config/instance_config.lua

```diff
@@ -1118,6 +1118,60 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    security = schema.record({
+        auth_type = schema.enum({
+            'chap-sha1',
+            'pap-sha256',
```

Comment:
`pap-sha256` is supported only on Tarantool Enterprise Edition and we should reflect it in the validation.

Reply:
Done.